src/main/java/seedu/findvisor/commons/util/DateTimeUtil.java

```diff
@@ -54,7 +54,7 @@ public static boolean isValidDateTime(String dateTime) {
 
     /**
      * Converts a String into a LocalDateTime object. The expected format is dd-MM-yyyy'T'HH:mm. For
```

Comment:
```suggestion
     * Converts a String into a LocalDateTime object. The expected format is dd-MM-uuuu'T'HH:mm. For
```
Would this change be better to match the exact date pattern in the code?

Reply:
I think changing the code to fit the output format would be better, although I am not sure if this is a feature change. This is because `uuuu` does not seem like an intuitive symbol that is used for year, even though it is 4 digits long. I am more inclined to keep the current date pattern for output message as it is clearer.

Reply:
I'll change it to `uuuu` to match what's being used in the code, the example given should help to clear up what is `uuuu` for those that don't know.